velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1041,4 +1042,33 @@ struct ParseDateTimeFunction {
   }
 };
 
+template <typename T>
+struct CurrentTimeFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE bool call(
+      out_type<Varchar>& result) {
+
+    using namespace std::chrono;
+
+    auto now = system_clock::now();
```

Comment:
Will try that out, thanks!

Reply:
@pedroerp I have changed the implementation to use `Timestamp::now()` and also the DateTimeFormatter that is in Velox

Reply:
@minhancao : It might be cleaner to add a function in Timestamp to print out the iso8601 string representation with timezone and invoke that with Timestamp::now().

I also didn't understand the purpose of the formatter here.